app/model/sdl/Abstract/data.js

```diff
@@ -1231,6 +1231,30 @@ SDL.SDLModelData = Em.Object.create(
                 "imageSupported": true,
                 "textSupported": true
               }],
+              "keyboardCapabilities":{
```

Comment:
```suggestion
              "keyboardCapabilities": {
```

Reply:
Please add extra space after `:` symbol in each line

Reply:
Fixed in 718d529d03402e216d1b77766a4b819e0a00ebaf
Please take a look.